### PR TITLE
Metadata documentation

### PR DESCRIPTION
**Pull request by unnamedd**
In this PR I've added:
- the new metatags in the template files
- templates folder into the blocked list
- update README adding new metatags

### CODE REVIEW

File: Tools/Toolkit/Sources/ToolkitLibrary/Toolkit.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ public final class Toolkit {
 
   public func generateDocumentation(
     outputFileName: String = "extensions",
-    blockedFolderList: [String] = [".git", "screenshots", "Tools", ".build", ".github"]
+    blockedFolderList: [String] = [".git", "screenshots", "Tools", ".build", ".github", "templates"]
```
Comment (thomaspaulmann) [inline Tools/Toolkit/Sources/ToolkitLibrary/Toolkit.swift:20]: Neat!

File: templates/script-command.template.sh [hunk 2]
```diff
@@ -15,5 +15,10 @@
 # @raycast.currentDirectoryPath ~
 # @raycast.packageName Raycast Scripts
 # @raycast.needsConfirmation false
+# @raycast.description Write a nice and descriptive summary about your script command here
+# 
+# Author information:
+# @raycast.author Your name
+# @raycast.authorURL An URL for one of your social medias
```
Comment (thomaspaulmann) [inline templates/script-command.template.sh:22]:
```suggestion
#
# Documentation:
# @raycast.description Write a nice and descriptive summary about your script command here 
# @raycast.author Your name
# @raycast.authorURL An URL for one of your social medias
```

Comment (thomaspaulmann) [reply templates/script-command.template.sh:22]: Let's separate the documentation specific parameters.

File: templates/script-command.template.swift [hunk 3]
```diff
@@ -15,5 +15,11 @@
 // @raycast.currentDirectoryPath ~
 // @raycast.packageName Raycast Scripts
 // @raycast.needsConfirmation false
+// @raycast.description Write a nice and descriptive summary about your script command here
+// 
+// Author information:
+// @raycast.author Your name
+// @raycast.authorURL An URL for one of your social medias
+
```
Comment (thomaspaulmann) [inline templates/script-command.template.swift:23]: Same here.
```suggestion
//
// Documentation:
// @raycast.description Write a nice and descriptive summary about your script command here
// @raycast.author Your name
// @raycast.authorURL An URL for one of your social medias

```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Let's separate the parameters that are required for documentation purposes in the template.

**Comment by unnamedd** [comment]
All changes done @thomaspaulmann 🤘

**Comment by thomaspaulmann** [comment]
Thanks for addressing the changes!

**Pull request merged**